src/flags.rs

```diff
@@ -177,6 +188,7 @@ impl FlagContext {
 
         Ok(())
     }
+
```

Comment:
Chris and I both have our editors set up to run `rustfmt` on save, which would remove extra whitespace like this one. You can use `#[rustfmt::skip]` wherever you don't want the formatter to run